.github/workflows/build-pr.yml

```diff
@@ -64,7 +64,7 @@ jobs:
   build_test_e2e:
     name: "Build and run e2e Test"
     runs-on: ubuntu-latest
-    timeout-minutes: 25
+    timeout-minutes: 35
```

Comment:
curious, why does the test take so long now? Does rego check take much longer?

Reply:
Now I'll let local e2e test run twice, one for config policy, the other for rego policy, which would take longer than 25min.